terasoluna-gfw-functionaltest-web/pom.xml

```diff
@@ -131,9 +116,22 @@
     <!-- == Begin Tomcat == -->
     <!-- Servlet API 4.0 & JSP API 2.3 -->
```

Comment:
Servlet 4.0 -> Servlet 6.0
JSP 2.3 -> JSP 3.1

Reply:
Fixed according to review comments
fb66dc4